addons/mail/static/src/core/common/chat_window.xml

```diff
@@ -96,7 +96,7 @@
                 />
             </t>
         </div>
-        <div t-if="!props.chatWindow.folded or ui.isSmall" class="d-flex flex-column h-100 overflow-auto position-relative bg-inherit" t-att-class="{ 'opacity-50': state.editingName }" t-ref="content">
+        <div t-if="!props.chatWindow.folded or ui.isSmall" class="d-flex flex-column h-100 overflow-auto position-relative bg-inherit" t-att-class="{ 'opacity-50': state.editingName }" t-on-click="onClickContent" t-ref="content">
```

Comment:
`onClickContent` here works as long as the click on link doesn't stop propagation.
That's the case now but I wouldn't trust we keep it that way.

A better solution is to put the logic of `toggleFold` in the `store.handleClickOnLink`, which is intended as the handle method for all links in discuss.

`handleClickOnLink` has param `ev` and `thread`. Presence of chat window of thread doesn't mean the click was made in chat window (could be made in chatter!), so would use `ev.target.closest(".o-mail-ChatWindow")` as something alike `env.inChatWindow` without making any changes to method signature (this is FIX in stable so preferred).